data/hai/hai reveal 4 middle.txt

```diff
@@ -966,8 +966,35 @@ mission "Hai Reveal [A08] Symbol Identification"
 		log "Giti identified the angry cat symbol. It is from a long-defunct subversive group within the Syndicate called Base Principles. Somehow they obtained alien technology, and used the knowledge gained from it to advance their influence within the organization. A half-century ago, Syndicate Internal Affairs scoured them from the Syndicate's ranks in an extremely thorough and prejudiced inquisition."
 		
 		conversation
-			`After landing, you search up the way to the adjunct research library at the Navy Intelligence Observatory in Moscow. It takes three shuttle journeys and one high-speed rail trip, but you reach the building in under three hours. For someone of your background there are parts of this trip that seem dizzyingly unreal. As well traveled as you now are, actually navigating the urban environments of <origin> away from any particular spaceport is an experience not quite like any other. Once you reach the building in question you take a moment at the reception desk to interpret 'room number: 3017A19' - the 301st floor, section 7A, room 19 - and get directions. It's amazing how large Earth's megabuildings have become. Some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`
+			`You land at Ulaanbaatar's spaceport and receive a message that Batu has to finalize some arrangements and will arrive in the evening. That leaves plenty of time for you to see to your other business about the symbol.`
+			`	At an information desk, you inquire about the route to Moscow, and are given two options. You can take a very expensive private taxi service, which you could bill to Alondo. Or, you can take the long route and see the sights. Either way, you'll have plenty of time before Batu arrives.`
+			`	The attendant at the information desk asks you, "Would you like to contact the direct taxi service, or take the slow route and see the sights?"`
+			choice
+				`	"I'm in a hurry. I should take the direct route."`
+					goto "take the direct route"
+				`	"The slow route sounds more scenic. I'll try that."`
+			`	Moscow is thousands of kilometers away, so you travel most of the route on shuttlecraft. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but it would be too dull to be restrained to just one planet.`
+			`	You fly past mountain ranges and lakes, and over bare tundras and steppes. The gray cities of Earth stand out when you pass them; packed in their ancient locations and their new ones. Even from far above, the cities' vast megabuildings are an imposing sight. There's still some farmland on this route, even now that farming has largely moved off-world; but as you well know, all of that isn't enough for the twenty billion people living on Earth.`
```

Comment:
```suggestion
			`	You fly past mountain ranges and lakes, and over bare tundras and steppes. The gray cities of Earth stand out when you pass them; packed in their ancient locations and their new ones. Even from far above, the cities' vast megabuildings are an imposing sight. There's still some farmland on this route, even now that farming has largely moved off-world, because, as you well know, all of that isn't enough for the twenty billion people living on Earth.`
```

This feels a lot smoother a way to split this up, but I'm not entirely sure it's appropriate anyway, as an assumption of what player "well knows."

Reply:
I changed it, so it doesn't tell the player that they understand the food economics of the Republic. (No more "as you well know.")